### PR TITLE
Info-components

### PR DESCRIPTION
**Pull request by Mati0x**
#306

### CODE REVIEW

**Comment by Corantin** [comment]
Also ad it there if possible
image

File: apps/web/components/InfoBox.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  classNames,
+  icon,
+  isCapitalize = false,
+}: InfoBoxProps): JSX.Element {
+  return (
+    <div
+      className={`${BASE_STYLES} ${infoBoxStyles[infoBoxType]} ${classNames}`}
+    >
+      <div className="h-9 w-9 text-inherit">
+        {" "}
```
Comment (Corantin) [inline apps/web/components/InfoBox.tsx:38]:
```suggestion
```

File: apps/web/components/InfoBox.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        {icon ?? <InformationCircleIcon className=" h-9 w-9" />}
```
Comment (Corantin) [inline apps/web/components/InfoBox.tsx:39]: maybe keep no icon if someone not specifying it, that could be a potential need

File: apps/web/components/InfoBox.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      </div>
+      <p className="leading-5 text-neutral-content">
+        {isCapitalize ? content : capitalize(content ?? "")}
```
Comment (Corantin) [inline apps/web/components/InfoBox.tsx:42]: reverse ?

**Review by Corantin — APPROVED** [review]
Very nice, just small changes requested 🚀

File: apps/web/components/InfoBox.tsx (hunk 3 above)
Comment (Mati0x) [reply apps/web/components/InfoBox.tsx:42]: Ohh yeah yeah!

**Review by Corantin — APPROVED** [review]

**Review by Lucianosc — APPROVED** [review]
LGTM

**Pull request merged**